lib/resources/r4/related_person_patch.yaml

```diff
@@ -131,6 +142,44 @@ operations:
         <li>Requires a test operation to be provided for the `Relationship.id` field of the relationship to be removed.</li>
       </ul>
 
+  - name: replace-relationship-extensions
+    path: /relationship/{index}/extension
+    operation: replace
+    type: string
+    url: https://hl7.org/fhir/r4/related-person-definitions.html#RelatedPerson.relationship
```

Comment:
Does not look like the link should be working, because the resource field definition exists at:
`https://hl7.org/fhir/r4/relatedperson-definitions.html#RelatedPerson.relationship`

Probably it's just wrong in too many other places.

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/662/commits/f940696cabd6f27c86a9e0b4b1e62676f834e0ed#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dR149